### PR TITLE
FEAT-#6389: Ignore logs.

### PR DESCRIPTION
**Pull request by rentruewang**
This commit creates a gitignore file in the 🪵 directory. Closes: #6389.

## What do these changes do?

Creates a gitignore file with `*` as content, to prevent addition of logs to git repos when using modin in git repositories. Inspired by `pytype`.

- [x] first commit message and PR title follow format outlined [here](https://modin.readthedocs.io/en/latest/development/contributing.html#commit-message-formatting)
  > **_NOTE:_**  If you edit the PR title to match this format, you need to add another commit (even if it's empty) or amend your last commit for the CI job that checks the PR title to pick up the new PR title.
- [x] passes `flake8 modin/ asv_bench/benchmarks scripts/doc_checker.py`
- [x] passes `black --check modin/ asv_bench/benchmarks scripts/doc_checker.py`
- [x] signed commit with `git commit -s` 
- [x] Resolves #? 
- [x] tests added and passing
- [x] module layout described at `docs/development/architecture.rst` is up-to-date

### CODE REVIEW

**Review by RehanSD — DISMISSED** [review]
Shouldn’t we just modify the .gitignore file to ignore everything inside .modin instead of doing it programmatically?

**Comment by rentruewang** [comment]
It's for the people using modin. E.g. when you have tests using modin, the .modin would get added to the git repository, which is usually never the intention.

**Comment by anmyachev** [comment]
Wouldn't it be better to move `.modin` folder to `tmp` folder?

**Comment by rentruewang** [comment]
@anmyachev Not my decision to make unfortunately. However, I don't think `/tmp` is the best way as there "might be" persistent cache etc in the `.modin` folder.

File: modin/logging/config.py [hunk 1]
```diff
@@ -117,8 +117,19 @@ def _create_logger(
     Logger
         Logger object configured per Modin settings.
     """
-    log_filename = f".modin/logs/job_{job_id}/{log_name}.log"
-    os.makedirs(os.path.dirname(log_filename), exist_ok=True)
+
```
Comment (anmyachev) [inline modin/logging/config.py:120]:
```suggestion
```

**Review by anmyachev — APPROVED** [review]
Since the cache is saved to the current working directory, I agree that this can get in the way and something needs to be done about it. The proposed approach suits me too.

**Review by vnlitvinov — DISMISSED** [review]
LGTM!

File: modin/logging/config.py (hunk 1 above)
Comment (rentruewang) [reply modin/logging/config.py:120]: With that being said, it's NBD and I'm fine either way.

Comment (rentruewang) [reply modin/logging/config.py:120]: I see. It's one of the checks. My bad.

**Comment by anmyachev** [comment]
Thanks @rentruewang!

**Pull request merged**